UPGRADE-3.x.md

```diff
@@ -0,0 +1,8 @@
+UPGRADE 3.x
+===========
+
+### Tests
+
+All files under the ``Tests`` directory are now correctly handled as internal test classes. 
```

Comment:
Plus, you put this on 3.x and targeted the master branch, this is a non sense..

@greg0ire I think this should be reverted on master and then applied as PR on 3.x.

Even if you have a proper way...